store/storetest/channel_store.go

```diff
@@ -394,7 +394,10 @@ func testChannelStoreGet(t *testing.T, ss store.Store, s SqlSupplier) {
 	m2.UserId = u2.Id
 	m2.NotifyProps = model.GetDefaultChannelNotifyProps()
 
-	store.Must(ss.Channel().SaveDirectChannel(&o2, &m1, &m2))
+	if _, err := ss.Channel().SaveDirectChannel(&o2, &m1, &m2); err != nil {
```

Comment:
We prefer to user here something like:
```go
_, err := ss.Channel().SaveDirectChannel(&o2, &m1, &m2)
require.Nil(t, err)
```

Reply:
And of course the same for all the other cases where you just put a `panic` there.

Reply:
Done. 
@jespino, thanks for the review :)